Controls/CAN.csv

```diff
@@ -0,0 +1 @@
+0x221,20485
```

Comment:
This CSV file should also be in the gitignore. No need to track it since it'll cause unnecessary merge conflicts